docs/lib/auth/fragments/flutter/signin/40_multi_factor_signup.md

```diff
@@ -1,22 +1,22 @@
 ```dart
 try {
   Map<String, dynamic> userAttributes = {
-    "email": "email@domain.com",
+    'email': 'email@domain.com',
     // Note: phone_number requires country code
-    "phone_number": "+15551234",
+    'phone_number': '+15551234',
```

Comment:
nit: maybe make this a full 10-digit US phone number since you're prepending with the US country code

Reply:
@palpatim I should absolutely do this - good catch.